### PR TITLE
README: Detail the submitters steps

### PR DESCRIPTION
**Pull request by Olf0**

### CODE REVIEW

**Comment by Olf0** [comment]
… now that I went through this once.

File: README.md
Comment (rinigus) [inline README.md:88]: In practice, it is sufficient just to submit package by specifying "sailfishos:chum". No need for version or anything else unless you plan to change version immediately. Package name stays the same anyway, so no need to specify it either.

**Review by rinigus — CHANGES_REQUESTED** [review]
One comment, as shown below. Please review and I would suggest to use simpler procedure in our description to avoid developers spending too much time on submission

File: README.md
Comment (piggz) [inline README.md:77]: I had to look up the meaning of extant!

Comment (Olf0) [reply README.md:77]: Seriously!?!  Oops, wow, well, it is shorter and supposedly "higher level language" than "existing", which can be ambiguous (depends on the context).  Would you prefer "existing" over "extant"?
I used "third party" further down, which IMO is not perfect either.

File: README.md
Comment (Olf0) [reply README.md:88]: Thank you @rinigus, this is exactly the guidance I needed; both, in practice ("What do I have to insert into these fields, are they mandatory or optional, and which scope does the description have?") and for enhancing this guide.

I will rephrase it using this information.

File: README.md
Comment (piggz) [reply README.md:77]: Or current?, or just "...tools and libraries... ?  We could keep extant, as you say, its well defined, but if a native like myself had to look it up, it may be confusing!

Comment (Olf0) [reply README.md:77]: > Or current?, or just "...tools and libraries... ?

IMO "current" does not really fit and leaving it out does also not suit this well, because both do not express that these "tools and libraries" are maintained elsewhere and are not originally written for SailfishOS.  In short, that an "upstream repo" exists, which is not under control of the submitter (thus I used "third party" further down).

> We could keep extant, as you say, its well defined, but if a native like myself had to look it up, it may be confusing!

While I try refrain from using "over-the-top language and grammar", because that does create a barrier for people to comprehend the content, and one may perceive "extant" being close to this realm, I also try hard to avoid ambiguities, colloquialisms, using deliberately simple language (where "do" and "get" are the ~~ubiquitous~~ omnipresent verbs, but also the meaning becomes hard to grasp) etc.  Hence I like "extant" and prefer to keep it, because it is concise.

Writing documentation and specifications for an international audience always needs to balance expressiveness against simplicity.  While I tend to prefer expressiveness, I deliberately wrote "~~ubiquitous~~ omnipresent" above, because they mean exactly the same, but I perceive "ubiquitous" as a modern day buzzword and "over-the-top language".

File: README.md
Comment (Olf0) [reply README.md]: I hope [commit ff11eae](https://github.com/sailfishos-chum/main/pull/68/commits/ff11eae9f238a328c88597c726be52f42c5b8ed1) does address this well.

File: README.md
Comment (rinigus) [reply README.md:77]: please drop "extant". It doesn't help that we have to look it up to read README

Comment (Olf0) [reply README.md]: Done, see [commit d73de84](https://github.com/sailfishos-chum/main/pull/68/commits/d73de84f1c8a69b3d394ed68c52a83ee2cbbba47).

Comment (rinigus) [reply README.md]: Thank you! @piggz - would you finalize this PR? It sounded like you reviewed it already

File: README.md
Comment (Olf0) [reply README.md]: @rinigus, this review comment thread you started is still shown as "Changes requested" below (right above empty comment field at the very bottom), maybe this is why @piggz does not approve this PR.
When I hit the "Resolved" button for this comment thread, that does not change this, I assume somebody with maintainer's rights need to do that.

Hence I re-requested your review "formally".

**Comment by rinigus** [comment]
LGTM, merging

**Pull request merged**
